com.geekbench.Geekbench6.yml

```diff
@@ -0,0 +1,31 @@
+app-id: com.geekbench.Geekbench6
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: geekbench6
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: geekbench
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/bin
+      - cp -r geekbench/* ${FLATPAK_DEST}/bin/
+      - install -Dm755 geekbench/geekbench6 ${FLATPAK_DEST}/geekbench6
+      - install -Dm644 app-icon.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dm644 com.geekbench.Geekbench6.appdata.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.appdata.xml
+    sources:
+      - type: file
+        path: app-icon.svg
+      - type: file
+        path: com.geekbench.Geekbench6.appdata.xml
+      - type: archive
+        url: https://cdn.geekbench.com/Geekbench-6.1.0-Linux.tar.gz
```

Comment:
how does it work on aarch64?

Reply:
I don't know and I don't think I have the means to test.

Reply:
unsually when some blob is shipped marked as "Linux" the chances it run on anything else but x86_64 are "unlikely". But still an aarch64 flatpak is built.

Reply:
Safest is probably disabling. There are three binaries, from what I can tell.

`geekbench6`
`geekbench_avx2`
`geekbench_x86_64`